src/gpm-statistics.c

```diff
@@ -431,7 +431,7 @@ gpm_stats_update_info_page_details (UpDevice *device)
 		/* TRANSLATORS: Present is whether the device is currently attached
 		 * to the computer, as some devices (e.g. laptop batteries) can
 		 * be removed, but still observed as devices on the system */
-		gpm_stats_add_info_data (_("Present"), gpm_stats_bool_to_string (is_present));
```

Comment:
This change should be removed, or else it breaks presence status. "State" is already handled several lines below.